install/kubernetes/helm/istio/values-istio-auth-galley.yaml

```diff
@@ -0,0 +1,29 @@
+# This is used to generate istio.yaml
```

Comment:
I don't think this is a good practice - in general users should do '--set galey.enabled:true'. The other settings in this file would override or be overridden by other values (if user tries to use multiple values). 

For test purpose - use set as well.

Reply:
AFAICT, there is no straightforward way for a test to specify prerequisite dependencies. The test itself could generate the final install YAML from helm templates using custom values. But that requires refactoring the test framework, which is outside the scope of this PR. I believe that problem is being addressed more systematically by testing SWAT team cc @hklai @ozevren @nmittler

Reply:
We are improving local testing from two sides.
1. Making it easier to bring up cluster locally to run E2E tests. That has no conflict with what this PR is trying to do.
2. Implement a framework that can start up local processes/in memory instances for faster testing. @ozevren and @nmittler are working on this right now. I suppose it can be extended to include setups like this in future.